src/test/java/org/springframework/data/elasticsearch/junit/jupiter/SpringDataElasticsearchTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.elasticsearch.junit.jupiter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+/**
+ * Combines the {@link SpringDataElasticsearchExtension} and the {@link SpringExtension}.
+ *
+ * @author Peter-Josef Meisch
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@ExtendWith(SpringDataElasticsearchExtension.class)
+@ExtendWith(SpringExtension.class)
+public @interface SpringDataElasticsearchTest {
```

Comment:
I'm not sure whether we need this, but how about renaming this annotation to `@IntegrationTest`? It probably makes sense to remove `@Extend(SpringExtension)` here and introduce it in Spring-specific tests. The reason is that `SpringDataElasticsearchExtension` should handle primarily the Elasticsearch connectivity aspect. The integration with Spring is a secondary concern. See also comments on `SpringDataElasticsearchExtension`.

Reply:
The reason I created this annotation was to combine the two extensions in the right order, so that `SpringDataElasticsearchExtension` runs before `SpringExtension. One can still use the extensions on their own wit `@ExtendWith` like I did in the new `JUnit5ClusterConnectionTests`test

Reply:
Then let's shoot for `@SpringIntegrationTest` and `@IntegrationTest` annotations for easier consumption in the tests.

Reply:
done that.